src/pages/Wallet/AddressDetailsPage.tsx

```diff
@@ -140,63 +128,19 @@ const AddressDetailsPage = () => {
         </DataListRow>
       </DataList>
       <PageH2>{t`Transaction history`}</PageH2>
-      <Table headers={transactionsTableHeaders} minWidth="500px">
+      <Table minWidth="500px">
         {address.transactions.pending
           .slice(0)
           .reverse()
-          .map(({ txId, timestamp, toAddress, amount, type }) => (
-            <TableRow key={txId} columnWidths={tableColumnWidths} blinking>
-              <TableCell>
-                <TransactionalInfo content={t`Pending`} type="pending" />
-              </TableCell>
-              <TableCell>{dayjs(timestamp).fromNow()}</TableCell>
-              <TableCell>
-                <DirectionalAddress>
-                  <DirectionBadge>{t`To`}</DirectionBadge>
-                  <Truncate>{toAddress}</Truncate>
-                </DirectionalAddress>
-              </TableCell>
-              <TableCell align="end">
-                {type === 'transfer' && amount && <TransactionalInfo type="out" prefix="-" content={amount} amount />}
-              </TableCell>
+          .map((transaction) => (
+            <TableRow key={transaction.txId} blinking>
+              {transaction.type === 'transfer' && <TransactionalInfo hideLabel transaction={transaction} />}
             </TableRow>
           ))}
         {address.transactions.confirmed.map((transaction) => {
-          const amount = calAmountDelta(transaction, addressHash)
-          const amountIsBigInt = typeof amount === 'bigint'
-          const isOut = amountIsBigInt && amount < 0
-
           return (
-            <TableRow
-              key={transaction.hash}
-              columnWidths={tableColumnWidths}
-              onClick={() => onTransactionClick(transaction)}
-            >
-              <TableCell>
-                <TransactionalInfo content={isOut ? '↑ ' + t`Sent` : '↓ ' + t`Received`} type={isOut ? 'out' : 'in'} />
-              </TableCell>
-              <TableCell>{dayjs(transaction.timestamp).fromNow()}</TableCell>
-              <TableCell>
-                <DirectionalAddress>
-                  <DirectionBadge>{isOut ? t`To` : t`From`}</DirectionBadge>
```

Comment:
Remove both from App.json

Reply:
These are still used in the transaction details modal

Reply:
Aren't they with lower case there? I think it's important to distringuish.